src/routes/docs/advanced/self-hosting/sites/+page.markdoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: article
+title: Sites
+description: Harness the full power of self-hosted sites with Appwrite. Explore site deployment, management, and integration in your self-hosted environment.
+---
+
+{% partial file="configure-github-app.md" /%}
+
+# Configure Sites runtimes {% #sites %}
+
+Not all site runtimes are enabled by default. Enable the runtimes that you need and disable unused runtimes to save disk space on your server.
+To enable a runtime, add it to the `_APP_SITES_RUNTIMES` environment variable as a comma-separated list.
+
+The three runtimes currently available for Sites are the Static, Node.js 22, and Flutter 3.29 runtimes.
+
+```bash
+_APP_SITES_RUNTIMES=static-1,node-22,flutter-3.29
+```
+
+You can also configure the maximum timeout that can be set on individual Appwrite Sites. The maximum configurable timeout can be increased by changing the `_APP_SITES_TIMEOUT` environment variable. This environment variable changes the configurable maximum but does not alter existing configurations of individual sites.
+
+{% partial file="update-variables.md" /%}
+
+# Add an apex domain for Sites {% #root-domain %}
+
+Appwrite allows you to add an apex domain to your instance's configuration that you can publicly expose your Site on.
+To add an apex domain, you must first configure either one of the following environment variables in your `.env` file:
+
+- `_APP_DOMAIN_TARGET_A`: Set this to the IPv4 address of your server.
+- `_APP_DOMAIN_TARGET_AAAA`: Set this to the IPv6 address of your server.
+
+Head to the **Domains** tab of your site, and add your apex domain (e.g., `myapp.com`) as a new domain pointed to the active deployment.
+
+Next, head to your DNS provider and create an A or AAAA record for your root domain pointing to the server's IP address. This will allow Appwrite to serve your sites from the apex domain. 
+
+> Note: Make sure to remove any pre-existing values for `_APP_DOMAIN_TARGET_A` and `_APP_DOMAIN_TARGET_AAAA` if they are set, as these can conflict with your current configuration.
```

Comment:
Lets rewrite this to remove them from DNS settings. For example when I buy matejbaco.eu on NameCheap, they likely automatically point A or AAAA (or both) to their server, so website can say "Nothing is here yet".

Inagime now I switched A record to my Appwrite instance, it would still have AAAA pointong to NameCheap temporary error page, so depending if visitor uses IPv4 or IPv6, they might be getting my page, or error page.

That's why it's important to delete all A and AAAA from `@` (apex) DNS records.

Reply:
Okay